rpm/aardvark-dns.spec

```diff
@@ -53,6 +53,24 @@ BuildRequires: rust-srpm-macros
 Forwards other request to configured resolvers.
 Read more about configuration in `src/backend/mod.rs`.
 
+%package tests
+Summary: Tests for %{name}
+
+Requires: %{name} = %{epoch}:%{version}-%{release}
+Requires: bats
+Requires: bind-utils
+Requires: iptables
```

Comment:
the iptables should be dropped, the depency is set by netavark already

Reply:
ack. Updating.